.github/workflows/release-please.yml

```diff
@@ -26,6 +26,22 @@ jobs:
         with:
           fetch-depth: 0 # If you only need the current version keep this.
 
+      - name: Update launchdarkly-server-sdk rockspec
+        uses: ./.github/actions/update-rockspec
+        if : ${{ steps.release.outputs.prs_created == 'true' }}
```

Comment:
For this to work for me, I had to bump release please to v4. Which lead to some other changes too. Probably worth really testing these changes before merging.

Reply:
Ok, I've changed tactics.

I added a `manual-update-versions` workflow instead of modifying `release-please` workflow. This way I can select a branch and test it out in Github UI before messing with the current release process.

Additionally, I've tested `manual-update-versions` with `act` locally and it appears to succeed: https://github.com/launchdarkly/lua-server-sdk/commits/casey-test/

Once that's verified, I'll put up another PR to add it into the release-please flow with the v4 update.